server/auth/BitsAuth0Provider.ts

```diff
@@ -6,7 +6,8 @@ const BitsAuth0Provider = Providers.Auth0({
   clientId: process.env.AUTH0_CLIENT_ID as string,
   clientSecret: process.env.AUTH0_CLIENT_SECRET as string,
   domain: process.env.AUTH0_DOMAIN as string,
-});
+  authorizationUrl: `https://${process.env.AUTH0_DOMAIN}/authorize?response_type=code&prompt=login`,
+} as any); // 'as any' to pass the new "authorizationUrl" to the auth provider. This will always prompt for login from the auth provider (for after signing out)
```

Comment:
This works, however now you can login when you are already logged in. This isn't a big deal because #103 will make sure you can't reach the login page if you are already logged in.